README.md

```diff
@@ -5,6 +5,10 @@
 
```

Comment:
- [ ] Since this is a group project, it will be nice to update the `Authors Section` of your readme file with information pertaining to the other team member, as this will make your readme file more informative and professional.

Reply:
On it!